proto/cosmos/gov/v1beta1/tx.proto

```diff
@@ -91,3 +92,15 @@ message MsgDeposit {
 
 // MsgDepositResponse defines the Msg/Deposit response type.
 message MsgDepositResponse {}
+
+// MsgSignal is a simple text-based message that can be added to a proposal if
+// the proposer wants to signal a change. Nothing is executed.
+message MsgSignal {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer)         = false;
+  option (gogoproto.goproto_getters)  = false;
+
+  string title       = 1;
+  string description = 2;
+}
```

Comment:
Maybe we just add `title` & `description` fields to `MsgSubmitProposal`?

Reply:
That's the alternative to adding a signal message in the proposal. I wasn't sure if all proposals necessary needed a title and a description. In some cases a link/address that points to the forum / proposal write up might suffice

Reply:
I think a link is more common and maybe a better idea. We should discuss. I don't think the title and description metadata should extend to group proposals for a number of reasons. Some groups may be want to be more anonymous and including title and description could confuse users into sharing private info on a public blockchain.